tests/test_callbacks.py

```diff
@@ -15,51 +15,72 @@
 
 
 @mark.parametrize(
-    "app_path,args,expected",
+    "args,expected",
     [
         (
-            "tests/test_apps/app_with_callbacks/custom_callback/my_app.py",
             [],
             dedent(
                 """\
-                [HYDRA] Init CustomCallback with self.param1=value1, self.param2=value2
-                [HYDRA] CustomCallback on_run_start
-                [HYDRA] Init CustomCallback with self.param1=value1, self.param2=value2
-                [JOB] CustomCallback on_job_start
+                [HYDRA] Init custom_callback
+                [HYDRA] custom_callback on_run_start
+                [HYDRA] Init custom_callback
+                [JOB] custom_callback on_job_start
                 foo: bar
 
-                [JOB] CustomCallback on_job_end
-                [JOB] CustomCallback on_run_end"""
+                [JOB] custom_callback on_job_end
```

Comment:
You can use logger to print the config as well.

Reply:
done! :)